QuizApp/App/Screens/Leaderboard/LeaderboardCell.swift

```diff
@@ -0,0 +1,82 @@
+import UIKit
+
+final class LeaderboardCell: UITableViewCell {
+
+    static let reuseIdentifier = String(describing: LeaderboardCell.self)
+
+    private var nameLabel: UILabel!
+    private var pointsLabel: UILabel!
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+        createViews()
+        styleViews()
+        defineLayoutForViews()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func set(for leaderboard: LeaderboardModel, index: Int) {
+        let playerPosition = "\(index + 1). "
+        let attributedString = NSAttributedString(
+            string: playerPosition,
+            attributes: [
+                NSAttributedString.Key.font: Fonts.sourceSansProBold20.font,
+                NSAttributedString.Key.foregroundColor: UIColor.white
+            ]
+        )
+        let attributedLeaderboardString = NSAttributedString(string: leaderboard.name)
+
+        nameLabel.attributedText = combine(left: attributedString, right: attributedLeaderboardString)
+        pointsLabel.text = leaderboard.points
+    }
+
+}
+
+extension LeaderboardCell: ConstructViewsProtocol {
+
+    func createViews() {
+        nameLabel = UILabel()
+        addSubview(nameLabel)
+
+        pointsLabel = UILabel()
+        addSubview(pointsLabel)
+    }
+
+    func styleViews() {
+        backgroundColor = .clear
+
+        nameLabel.style(with: "test", color: .white, alignment: .left, font: Fonts.sourceSansProSemiBold18.font)
+
+        pointsLabel.style(with: "12345", color: .white, alignment: .left, font: Fonts.sourceSansProSemiBold18.font)
+    }
+
+    func defineLayoutForViews() {
+        nameLabel.snp.makeConstraints {
+            $0.top.equalToSuperview()
+            $0.leading.equalToSuperview().offset(20)
+            $0.bottom.equalToSuperview()
+        }
+
+        pointsLabel.snp.makeConstraints {
+            $0.top.equalToSuperview()
+            $0.trailing.equalToSuperview().inset(20)
+            $0.bottom.equalToSuperview()
+        }
+    }
```

Comment:
You're missing one constraint. If the name is too long it will look like this:
![Simulator Screen Shot - iPhone 14 - 2022-10-10 at 13 40 37](https://user-images.githubusercontent.com/9210723/194858379-e992228a-1be7-423c-9006-a889d6d551d3.png)